ICSharpCode.Decompiler/CSharp/ExpressionBuilder.cs

```diff
@@ -763,6 +763,21 @@ TranslatedExpression TranslateCeq(Comp inst, out bool negateOutput)
 				.WithRR(rr);
 		}
 
+		TranslatedExpression TryUniteEqualityOperandType(TranslatedExpression operand, TranslatedExpression otherOperand)
```

Comment:
Using names like `left` and `right` instead of `operand` and `otherOperand` will improve readability of the code.